CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [UNRELEASED]
+- Add `objc_class_prefix` option in managed mode, allowing `default`, `except` and `override`
+  in the same style as `optimize_for`.
```

Comment:
Instead of having this refer back to `optimize_for` changelog statement, we should have one that expands on this. We can use the same wording and structure as the previous changelog statement, but it's better if this was a standalone statement.